code/datums/abilities/wraith.dm

```diff
@@ -226,8 +227,10 @@
 
 		if (ishuman(T))
 			var/mob/wraith/W = holder.owner
-			return W.makeRevenant(T)
-			//return 0
+			. = W.makeRevenant(T)		//return 0
+			if(!.)
```

Comment:
why !

Reply:
Because for some reason the successful return is the negative if that makes sense. Without the !It plays the sound when it isn't successful. You can try it for yourself. I don't know why, it shouldn't act like this.

Reply:
it's called error code returning

Reply:
> it's called error code returning

well for some reason the error is what allows it to function. I don't understand it fully, but even though its awkward it seems functional and easily understandable.